src/main/target/STM32F7X2/config/NERO.config

```diff
@@ -0,0 +1,79 @@
+// Usual lawyer stuff here
```

Comment:
The `//` is probably not recognised as a comment in the CLI.

Reply:
😬